pkgs/development/python-modules/fdroidserver/default.nix

```
@@ -1,30 +1,72 @@
-{ stdenv
+{ androguard
+, Babel
 , buildPythonPackage
+, clint
+, defusedxml
+, docker
+, docker-py
 , fetchFromGitLab
+, GitPython
+, isPy3k
 , libcloud
-, pyyaml
+, mwclient
 , paramiko
+, pillow
 , pyasn1
 , pyasn1-modules
-, pillow
-, mwclient
-, GitPython
-, isPy3k
-}:
+, python
+, python-vagrant
+, pyyaml
+, qrcode
+, requests
+, ruamel_yaml
+, stdenv }:
 
 buildPythonPackage rec {
```

```suggestion
buildPythonApplication rec {
```
Isn't this an application rather than a module? If so, this should be changed and it should be moved out of `pythonPackages`.

---

I honestly don't know.

---

Do you use it? If so, as an application? If not, what do you think about simply dropping this package from nixpkgs?

---

I use it (which was actually why I added the nix expression, I'm not a masochist :D) as an application. I'm just not sure if one might use this as a library.

---

Fine with me, I'll change it.